helpers/query_setter_helper.go

```diff
@@ -0,0 +1,34 @@
+package helpers
+
+import (
+	"strings"
+)
+
+// BuildOrderByFor builds list of given order list from SQL format to Beego query setter format
+// ["user.name", "email desc"] --> ["user__name", "-email"]
+func BuildOrderByFor(oldOrderList []string) (newOrderList []string) {
```

Comment:
Minor naming preference for:

```suggestion
func FormatOrderByFor(sourceOrderParams []string) (formattedOrderParams []string) {
```

It would also be consistent with the what the method does with `formattedOrderBy`.